library/bootloader_settings.py

```diff
@@ -363,57 +365,39 @@ def run_module():
     for bootloader_setting in module.params["bootloader_settings"]:
         _unused, kernels_info, stderr = module.run_command("grubby --info=ALL")
         if "Permission denied" in stderr:
-            module.fail_json(msg="You must run this as sudo", **result)
+            module.fail_json("You must run this as sudo")
```

Comment:
I realize this is unrelated to the PR, but:

Is this message for the end user of the role?  If so, what action are they supposed to take?

Reply:
Run this as root or with ansible_become. I guess all our roles must be run with as root, but I don't see other modules to check for this. Anyway, I decided to add such a check.